SimDataFormats/Track/src/classes_def.xml

```diff
@@ -13,4 +13,11 @@
   <class name="edm::RefProd<std::vector<SimTrack> >"/>
   <class name="edm::Ref<std::vector<SimTrack>,SimTrack,edm::refhelper::FindUsingAdvance<std::vector<SimTrack>,SimTrack> >"/>
   <class name="edm::RefVector<std::vector<SimTrack>,SimTrack,edm::refhelper::FindUsingAdvance<std::vector<SimTrack>,SimTrack> >"/>
+  <class name="UniqueSimTrackId"/>
+  <class name="std::pair<unsigned int, EncodedEventId>"/>
+  <class name="std::pair<UniqueSimTrackId, edm::Ref<std::vector<TrackingParticle>, TrackingParticle, edm::refhelper::FindUsingAdvance<std::vector<TrackingParticle>, TrackingParticle> > >"/>
+  <class name="std::unordered_map<UniqueSimTrackId, TrackingParticleRef, UniqueSimTrackIdHash>"/>
+  <class name="edm::Wrapper<std::unordered_map<UniqueSimTrackId, TrackingParticleRef, UniqueSimTrackIdHash>>"/>
```

Comment:
The dictionaries for compound types depending on both `EncodedEventId` and `TrackingParticle` should be defined in the package where `TrackingParticle` is defined (as it already depends on `EncodedEventId`), i.e. `SimDataFormats/TrackingAnalysis`.